src/js/utils/api.js

```diff
@@ -41,25 +56,30 @@ export default class api {
   }
 
   /**
-   * @path 'notes', 'settings'
-   * { id } note id. String or (null or undefined)
-   *   All will be update if it was null or undefined
-   * { updates } updates Object
+   * put
+   *
+   * PUT
+   * @param {String} requestPath 'notes' or 'settings'
+   * @param {Object} { id, updates }
+   *   { id } note id. String or (null or undefined)
+   *     All will be update if it was null or undefined
+   *   { updates } updates Object
+   * @return {Promise}
    */
-  static put(path, { id, updates }) {
+  static put(requestPath, { id, updates }) {
```

Comment:
'destructuring expression' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).